src/modules/commentsApi.js

```diff
@@ -1,4 +1,4 @@
-const appId = 'QALIoXYe7LSsEgG2FYGA/';
+const appId = 'f5fDagMARo6BC6LKqLaP/';
```

Comment:
- [ ] Please add a if else statement to check if there is value in create comment input. Because I can add empty comment. 

![image](https://user-images.githubusercontent.com/69011963/172027031-49419268-1996-49bc-9b29-c664bc5039ad.png)